pepper-apis/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -665,6 +679,22 @@ public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
         }
     }
 
+    private static void createActivityInstance(Handle apisHandle, EventConfigurationDto eventConfDto, QueuedEventDto pendingEvent) {
+        User participant = apisHandle.attach(UserDao.class)
+                .findUserByGuid(pendingEvent.getParticipantGuid())
+                .orElse(null);
+        Optional<StudySettings> settings = apisHandle.attach(
+                StudyDao.class).findSettings(pendingEvent.getStudyGuid());
```

Comment:
It's unfortunate that we don't have the study_id already. For now, it might be better to grab the `StudyDto` from `JdbiUmbrellaStudy` since those are cached.